docs/en/ingest-management/commands.asciidoc

```diff
@@ -176,24 +176,26 @@ elastic-agent enroll --fleet-server-es <string>
                      [--fleet-server-cert <string>] <1>
                      [--fleet-server-cert-key <string>]
                      [--fleet-server-es-ca <string>]
+                     [--fleet-server-es-ca-trusted-fingerprint <string>] <2>
                      [--fleet-server-host <string>]
                      [--fleet-server-insecure-http]
                      [--fleet-server-policy <string>]
                      [--fleet-server-port <uint16>]
                      [--force]
                      [--help]
-                     [--url <string>] <2>
-                     [--fleet-server-es-insecure ] <3>
+                     [--url <string>] <3>
+                     [--fleet-server-es-insecure ]
                      [global-flags]
 ----
 <1> If no `fleet-server-cert*` flags are specified, {agent} auto-generates a
 self-signed certificate with the hostname of the machine. Remote {agent}s
 enrolling into a {fleet-server} with self-signed certificates must specify
 the `--insecure` flag.
-<2> Required when enrolling in a {fleet-server} with custom certificates. The
+<2> SHA256 fingerprint of the CA that {es} generates for self-signed certificates.
+Required when using self-signed certificate on {es} side.
```

Comment:
```suggestion
<2> Required when using self-signed certificates with {es}.
```

Reply:
I'd avoid providing a description in the callout. Otherwise, we'll end up with a bunch of callouts.